### PR TITLE
crashwatch-update-1.1.0

### PR DESCRIPTION
**Pull request by bedirhangull**

### CODE REVIEW

**Comment by ahmetb** [comment]
:robot: _Beep beep! I’m a robot speaking on behalf of @ahmetb._ :robot:

-----

This pull request **does not** seem to be a straightforward version bump. I'll have a human review this. If we don't respond within a day, feel free to ping us.

_Why wasn't this detected as a plugin version bump:_

>file `plugins/crashwatch.yaml` is not a straightforward version bump: could not parse version string `"v1.0.6\" "`
/kind plugin-update

File: plugins/crashwatch.yaml [hunk 1]
```diff
@@ -3,58 +3,101 @@ kind: Plugin
...
   homepage: https://github.com/bedirhangull/kubectl-crashwatch
   caveats: |
     Usage:
       $ kubectl crashwatch
-
     For additional options:
       $ kubectl crashwatch --help
-      or visit the documentation at https://github.com/bedirhangull/kubectl-crashwatch/blob/v1.0.6/README.md
-
+      or visit the documentation at https://github.com/bedirhangull/kubectl-crashwatch/blob/v1.1.0/README.md
```
Comment (ahmetb) [inline plugins/crashwatch.yaml:99]: Fyi this part is redundant

**Review by ahmetb — APPROVED** [review]

**Comment by ahmetb** [comment]
Also, please consider setting up [Krew release automation](https://krew.sigs.k8s.io/docs/developer-guide/release/automating-updates/) which helps you skip manually making updates to your Krew manifests on each new version and send a pull request. It’s a GitHub workflow bot that can run every time you push a new tag. Those PRs are automatically approved.

**Comment by bedirhangull** [comment]
@ahmetb Yes, I saw it. But when I tried to update my plugin, the bot didn't update my 256 section. I didn't understand how to update automatically. Sorry :( I will not make this mistake again.

**Comment by ahmetb** [comment]
Np. Please see how other plugins handle it you're unclear.

**Pull request merged**